service/routes.py

```diff
@@ -58,8 +58,11 @@ def list_customers():
     app.logger.info("Request for customer list")
     customers = []
     email = request.args.get("email")
+    name = request.args.get("name")
```

Comment:
the `Customer` model has a `first_name` attribute, not a `name` attribute. change the `"name"` (and possibly the variable name) to `"first_name"`